CesiumGltfReader/test/TestGltfReader.cpp

```diff
@@ -400,3 +400,15 @@ TEST_CASE("Can parse doubles with no fractions as integers") {
       options);
   CHECK(!result.warnings.empty());
 }
+
+TEST_CASE("KTX2_hacky_test") {
```

Comment:
Could use a better test, or at least a better name. :)

Reply:
The test doesn't seem too hacky to me (maybe the tested model is), so how about just giving it a better name?